packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -366,30 +369,25 @@ export default {
             try {
                 await this.loadConsumerDetails({ api: this.consumerApi, authToken: this.authToken });
                 this.$log.info('Consumer details fetched successfully', standardLogTags);
+                // debug
```

Comment:
I've added some debug code to assist with testing in QA and will remove and do a second pr once complete.

Reply:
Is this still to be removed?

Reply:
Not yet